src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -562,6 +561,10 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<CreateProperty Value="None" Condition="'$(AndroidLinkMode)' == 'SdkOnly' And '$(AndroidUseSharedRuntime)' == 'true'">
 		<Output TaskParameter="Value" PropertyName="AndroidLinkMode" />
 	</CreateProperty>
+	<Error Code="XA1011"
```

Comment:
The problem with this approach, as I understand it, is that it isn't localizable.  This should probably be in a Task.

Reply:
Sounds good. I wasn't sure whether to use this PR to start a new pattern of mandating logging from within tasks for warnings and errors for localizability, but it does seem reasonable to use this PR to start that, so I'll adjust it!

Reply:
I found out that dotnet/sdk has a mechanism for this where it defines custom warning and error tasks that take resource names and format arguments instead of plain text:

  * [`NETSdkWarning.cs`](https://github.com/dotnet/sdk/blob/56a9d827c8408f98ab27caceae2c8e1936281401/src/Tasks/Common/NETSdkWarning.cs)
  * [`NETSdkError.cs`](https://github.com/dotnet/sdk/blob/56a9d827c8408f98ab27caceae2c8e1936281401/src/Tasks/Common/NETSdkError.cs)
  * [`MessageBase.cs`](https://github.com/dotnet/sdk/blob/56a9d827c8408f98ab27caceae2c8e1936281401/src/Tasks/Common/MessageBase.cs)

Example use: [`Microsoft.NET.RuntimeIdentifierInference.targets`](https://github.com/dotnet/sdk/blob/56a9d827c8408f98ab27caceae2c8e1936281401/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets#L129-L134)

I'll tentatively plan to do a basic imitation of that approach for now. I'm guessing xamarin-android will be able to switch from that imitation to the upstream dotnet/sdk mechanism itself once the .NET 5 transition is complete.